Source/JavaScriptCore/llint/WebAssembly.asm

```diff
@@ -414,7 +414,10 @@ if not JSVALUE64
 end
 
     loadp Callee[cfr], ws0
-    andp ~3, ws0
+    andp ~(constexpr JSValue::WasmTag), ws0
+    leap WTFConfig + constexpr WTF::offsetOfWTFConfigLowestAccessibleAddress, ws1
+    loadp [ws1], ws1
+    addp ws1, ws0
```

Comment:
Is it worth all these extra instructions?

Reply:
oh I guess the JITs get to avoid this.

Reply:
Yes, in JIT, everything is done at compile time.